packages/validator/src/services/block.ts

```diff
@@ -13,8 +13,13 @@ import {BlockDutiesService, GENESIS_SLOT} from "./blockDuties.js";
 type BlockProposingServiceOpts = {
   graffiti?: string;
   strictFeeRecipientCheck?: boolean;
+  builder: {enabled?: boolean};
 };
 
+type WrappedProduceBlockType =
+  | {type: allForks.BlockType.Full; data: allForks.FullOrBlindedBeaconBlock<allForks.BlockType.Full>}
+  | {type: allForks.BlockType.Blinded; data: allForks.FullOrBlindedBeaconBlock<allForks.BlockType.Blinded>};
```

Comment:
This `type` is from the spec or a wrapper here for convenience?

Reply:
for convenience :slightly_smiling_face: